### ISSUE TITLE
address ARIA findings

### ISSUE THREAD

**Issue by wesley-dean-gsa** [GSA-TTS/tts.gsa.gov#196]
During a chat about scanning the tts.gsa.gov site in preparation for deployment, the following came up:

> the only finding i see for you now (I’m testing in staging) is ARIA hidden element must not be focusable or contain focusable elements.
> a bunch of links like
```HTML
<a class="direct-link" href="#opportunity-overview" aria-hidden="true"></a>
```
> I think the aria fix is `Focusable content should have tabindex="-1" or be removed from the DOM`

**Comment by wesley-dean-gsa** [comment]
I installed [@axe-core/cli](https://www.npmjs.com/package/@axe-core/cli) on my local system and ran:

```
$ PREFIX='https://federalist-a2423046-fe43-4e75-a2ef-2651e5e123ca.sites.pages.cloud.gov/preview/gsa-tts/tts.gsa.gov/narumigsa-patch-3' ; curl -s "${PREFIX}/sitemap.xml" | xq "\"$PREFIX\(.urlset.url[].loc)\""  | xargs -n50 axe --tags=wcag2a

```

The output looks like this:

```
Running axe-core 4.8.4 in chrome-headless

Testing https://federalist-a2423046-fe43-4e75-a2ef-2651e5e123ca.sites.pages.cloud.gov/preview/gsa-tts/tts.gsa.gov/narumigsa-patch-3/preview/gsa-tts/tts.gsa.gov/narumigsa-patch-3/404/ ... please wait, this may take a minute.
 0 violations found!

Testing https://federalist-a2423046-fe43-4e75-a2ef-2651e5e123ca.sites.pages.cloud.gov/preview/gsa-tts/tts.gsa.gov/narumigsa-patch-3/preview/gsa-tts/tts.gsa.gov/narumigsa-patch-3/SECURITY/ ... please wait, this may take a minute.
 0 violations found!

Testing https://federalist-a2423046-fe43-4e75-a2ef-2651e5e123ca.sites.pages.cloud.gov/preview/gsa-tts/tts.gsa.gov/narumigsa-patch-3/preview/gsa-tts/tts.gsa.gov/narumigsa-patch-3/about/ ... please wait, this may take a minute.
 0 violations found!

Testing https://federalist-a2423046-fe43-4e75-a2ef-2651e5e123ca.sites.pages.cloud.gov/preview/gsa-tts/tts.gsa.gov/narumigsa-patch-3/preview/gsa-tts/tts.gsa.gov/narumigsa-patch-3/admin/ ... please wait, this may take a minute.
 0 violations found!

Testing https://federalist-a2423046-fe43-4e75-a2ef-2651e5e123ca.sites.pages.cloud.gov/preview/gsa-tts/tts.gsa.gov/narumigsa-patch-3/preview/gsa-tts/tts.gsa.gov/narumigsa-patch-3/allstar/ ... please wait, this may take a minute.
 0 violations found!

Testing https://federalist-a2423046-fe43-4e75-a2ef-2651e5e123ca.sites.pages.cloud.gov/preview/gsa-tts/tts.gsa.gov/narumigsa-patch-3/preview/gsa-tts/tts.gsa.gov/narumigsa-patch-3/ ... please wait, this may take a minute.
 0 violations found!

Testing https://federalist-a2423046-fe43-4e75-a2ef-2651e5e123ca.sites.pages.cloud.gov/preview/gsa-tts/tts.gsa.gov/narumigsa-patch-3/preview/gsa-tts/tts.gsa.gov/narumigsa-patch-3/overview/ ... please wait, this may take a minute.
 0 violations found!

Testing https://federalist-a2423046-fe43-4e75-a2ef-2651e5e123ca.sites.pages.cloud.gov/preview/gsa-tts/tts.gsa.gov/narumigsa-patch-3/preview/gsa-tts/tts.gsa.gov/narumigsa-patch-3/join/about-the-dha/ ... please wait, this may take a minute.
 0 violations found!

Testing https://federalist-a2423046-fe43-4e75-a2ef-2651e5e123ca.sites.pages.cloud.gov/preview/gsa-tts/tts.gsa.gov/narumigsa-patch-3/preview/gsa-tts/tts.gsa.gov/narumigsa-patch-3/join/compensation-and-benefits/ ... please wait, this may take a minute.
 0 violations found!

Testing https://federalist-a2423046-fe43-4e75-a2ef-2651e5e123ca.sites.pages.cloud.gov/preview/gsa-tts/tts.gsa.gov/narumigsa-patch-3/preview/gsa-tts/tts.gsa.gov/narumigsa-patch-3/join/federal-job-posting/ ... please wait, this may take a minute.
 0 violations found!

Testing https://federalist-a2423046-fe43-4e75-a2ef-2651e5e123ca.sites.pages.cloud.gov/preview/gsa-tts/tts.gsa.gov/narumigsa-patch-3/preview/gsa-tts/tts.gsa.gov/narumigsa-patch-3/join/hiring-process/ ... please wait, this may take a minute.
 0 violations found!

Testing https://federalist-a2423046-fe43-4e75-a2ef-2651e5e123ca.sites.pages.cloud.gov/preview/gsa-tts/tts.gsa.gov/narumigsa-patch-3/preview/gsa-tts/tts.gsa.gov/narumigsa-patch-3/join/ ... please wait, this may take a minute.
 0 violations found!

Testing https://federalist-a2423046-fe43-4e75-a2ef-2651e5e123ca.sites.pages.cloud.gov/preview/gsa-tts/tts.gsa.gov/narumigsa-patch-3/preview/gsa-tts/tts.gsa.gov/narumigsa-patch-3/join/tts-designer/ ... please wait, this may take a minute.
 0 violations found!

Testing https://federalist-a2423046-fe43-4e75-a2ef-2651e5e123ca.sites.pages.cloud.gov/preview/gsa-tts/tts.gsa.gov/narumigsa-patch-3/preview/gsa-tts/tts.gsa.gov/narumigsa-patch-3/join/resume/ ... please wait, this may take a minute.
 0 violations found!

Testing https://federalist-a2423046-fe43-4e75-a2ef-2651e5e123ca.sites.pages.cloud.gov/preview/gsa-tts/tts.gsa.gov/narumigsa-patch-3/preview/gsa-tts/tts.gsa.gov/narumigsa-patch-3/join/rolling-hiring/ ... please wait, this may take a minute.
 0 violations found!

Testing https://federalist-a2423046-fe43-4e75-a2ef-2651e5e123ca.sites.pages.cloud.gov/preview/gsa-tts/tts.gsa.gov/narumigsa-patch-3/preview/gsa-tts/tts.gsa.gov/narumigsa-patch-3/services/ ... please wait, this may take a minute.
 0 violations found!

Testing https://federalist-a2423046-fe43-4e75-a2ef-2651e5e123ca.sites.pages.cloud.gov/preview/gsa-tts/tts.gsa.gov/narumigsa-patch-3/preview/gsa-tts/tts.gsa.gov/narumigsa-patch-3/services/people/ ... please wait, this may take a minute.
 0 violations found!

Testing https://federalist-a2423046-fe43-4e75-a2ef-2651e5e123ca.sites.pages.cloud.gov/preview/gsa-tts/tts.gsa.gov/narumigsa-patch-3/preview/gsa-tts/tts.gsa.gov/narumigsa-patch-3/services/platforms/ ... please wait, this may take a minute.
 0 violations found!

Testing https://federalist-a2423046-fe43-4e75-a2ef-2651e5e123ca.sites.pages.cloud.gov/preview/gsa-tts/tts.gsa.gov/narumigsa-patch-3/preview/gsa-tts/tts.gsa.gov/narumigsa-patch-3/services/products/ ... please wait, this may take a minute.
 0 violations found!
Testing complete of 19 pages

Please note that only 20% to 50% of all accessibility issues can automatically be detected. 
Manual testing is always required. For more information see:
https://dequeuniversity.com/curriculum/courses/testingmethods

```

This scans a fairly recent branch (two days old now?).

**Comment by wesley-dean-gsa** [comment]
the one-liner I provided pulls down a sitemap from the branch on Cloud.gov Pages (formerly known as Federalist), iterates through all of the links in the sitemap to prepend the base (`PREFIX`) to each one, and then runs `axe` on each of those pages in batches of 50 (there are only 19 pages; the pagination prevents potentially overloading `axe`)
